Examples/Tests/initial_distribution/inputs

```diff
@@ -22,6 +22,7 @@ warpx.cfl     = 1.e-8
 ############ PLASMA #############
 #################################
 particles.species_names = gaussian maxwell_boltzmann maxwell_juttner beam
+particles.rigid_injected_species = beam
```

Comment:
I've added this because we currently have some unused parameters (`zinject_plane`, `rigid_advance`, etc.).